test-requirements.txt

```diff
@@ -2,3 +2,8 @@ mock
 pyfakefs
 pytest
 pytest-cov
+flake8
+nose
+pep8
+nose-exclude
+nose
```

Comment:
if we are adding 5.1 tests so we need to add to the test requirements  also the installation of cloudify (from config)

Reply:
add requirements for the integration tests like:
https://github.com/cloudify-cosmo/cloudify-azure-plugin/blob/master/test-requirements.txt#L15